pkgs/development/python-modules/pylint-odoo/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, pylint
+, pylint-plugin-utils
+, pytestCheckHook
+, pythonOlder
+, pythonRelaxDepsHook
+, setuptools
+, validators
+}:
+buildPythonPackage rec {
+  pname = "pylint-odoo";
+  version = "9.0.5";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
```

Comment:
```suggestion
```

Reply:
There are however downstream overlays that recover those python versions, such as https://github.com/cachix/nixpkgs-python/.

I think keeping this marker here is helpful for those use cases, and it doesn't seem a big maintenance burden here, just like [upstream inlcudes it too](https://github.com/OCA/pylint-odoo/blob/5f1715872ae50b072d8e1c0d10a31a4d64f2f66a/setup.cfg#L10).